JS/Lesson27/goksel/editPost/edit-post.css

```diff
@@ -0,0 +1,49 @@
+*{
```

Comment:
Try to move styles that are applied to all 3 HTML files into its own css file (e.g. common.css). This would include styles for *, body, etc..
Then, you'll add to css files to your HTML files but you'd not need to repeat the code in every CSS files which increases scalability.